src/core/tasks.py

```diff
@@ -413,8 +413,13 @@ def get_base_components(self):
     def get_json_validators(self):
         for task in self.tasks:
             for component in task.get_base_components():
-                for json_validator in component.json_validators:
-                    yield from json_validator
+                if isinstance(component, list):
+                    for component1 in component:
```

Comment:
i'm not a massive fan of `component1` variable naming, although it's tricky as we don't know whether it's a row or a component until we start iterating (when the isinstance check can be run).

Are we sure we can't just do like we did here? https://github.com/unipartdigital/workflows_engine/blob/c270400c183dd8fddf1476fe53f5c6ba3095cba4/src/core/components.py#L554-L559 so remove the else block

Reply:
It works little different with validators so cannot make as in validators, made a change which for my opinion is smth better not much tbh :)